assignments/assignment11/task.js

```diff
@@ -0,0 +1,22 @@
+var Task = function (category, title, priority, estimate) {
+    this.category = category;
+    this.title = title;
+    this.priority = priority;
+    this.estimate = estimate;
+    this.spent = 0;
+    this.remaining = 1.5;
+    this.track = function (trackNum) {
```

Comment:
This means that we are creating new functions for every newly created task. Will increase the memory used by the app. Notice that it is expected that we will create many tasks but one task manager, that's why it is acceptable to not use prototype in the task manager, but not a good idea in the case of tasks.